apps/web/src/pages/PluginsOverlay.tsx

```
@@ -231,6 +365,236 @@ export function PluginsOverlay({
+          connected
+            ? (event) => {
+                if (event.key === "Enter" || event.key === " ") {
+                  event.preventDefault();
+                  openDetail(item);
+                }
+              }
+            : undefined
+        }
+        role={connected ? "button" : undefined}
+        tabIndex={connected ? 0 : undefined}
+      >
```

Biome (2.5.8)

[error] Unexpected event handler on static element.

(lint/a11y/noStaticElementInteractions)